openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetailPage.tsx

```diff
@@ -256,7 +232,7 @@ const PoliciesDetailPage = () => {
     (rule: Rule) => {
       return (
         <Dropdown
-          disabled={!policyPermission.EditAll}
+          disabled={!isAdminUser}
```

Comment:
If non admin users doesn't have access to view the page, which we restricted from GlobalSettingsRouter.
Why do we need disabled conditions ?
Also are we not following the approach where we don't have access we are the hiding the action buttons only ?

Reply:
Thanks @karanh37 , I will remove the disabled part and as only the admin user will have the page access so will not hide anything on condition.